application/src/test/java/com/sanction/lightning/resources/FacebookResourceTest.java

```diff
@@ -66,6 +63,29 @@ public void testGetUserWithNullUsername() {
 
   @Test
   @SuppressWarnings("unchecked")
+  public void testGetUserWithNullResponse() {
```

Comment:
Maybe all of these should be called `testGetUserWithNullRetrofitResponse` to make it more clear?